src/helpers/tests/resultSpec.spec.ts

```diff
@@ -193,3 +193,45 @@ describe('getStackingResultSpec', () => {
         expect(getStackingResultSpec(buckets, [measureSortItem])).toEqual(expectedResultSpec);
     });
 });
+
+describe('getGeneralResultSpec', () => {
+    const measure = {
```

Comment:
please use `MEASURE_1` imported above instead of creating new variable

Reply:
AF.IMeasure is different from VisualizationObject.IMeasure. This is why this is not MEASURE_1